osc/commandline.py

```diff
@@ -4615,6 +4615,19 @@ def do_commit(self, subcmd, opts, *args):
             print("or use the --noservice option")
             return 1
 
+    def _has_pending_requests(self, api, prj, pkg, force=False):
+        if force:
+            return False
+
+        rqlist = get_request_list(api, prj, pkg)
```

Comment:
This should be limited to certain types of request, e.g. new, review.

Reply:
That's the default: new, review, declined. I suppose "declined" doesn't need to be there.

Reply:
Yes, otherwise you will depend on a third party to revoke declined requests.